lib/tasks/ctgov.rake

```diff
@@ -34,6 +34,14 @@ namespace :studyfinder do
       Trial.import force: true
     end
 
+    task cleanup_strays: :environment do |t, args|
+      puts "Cleaning up stray trials"
+      connector = Connectors::Ctgov.new
+      trials = connector.cleanup_stray_trials
+      puts "Have un-published (system_ids): "
+      puts trials.map{ |e| " #{e.system_id}\n" }
```

Comment:
`connector.cleanup_stray_trials` returns the result of calling `update(visible: true` on stray trials.  Since `update` will return the record(s) regardless of whether the update actually succeeded, would you need to do any final checking in this rake task to ensure what you are printing actually includes successfully updated records?

There's only a single validation I saw for `Trial`, so the likelihood of the update failing is probably minimal.  And you aren't actually manipulating the returned data, so maybe this doesn't matter much.

Reply:
Valid point. Updated to use `update!` so we get an error alert if there's an issue.